Formula/geotoad.rb

```diff
@@ -0,0 +1,21 @@
+class Geotoad < Formula
+  desc "Query tool to query the geocaching.com website"
+  homepage "https://github.com/steve8x8/geotoad"
+  url "https://github.com/steve8x8/geotoad/archive/3.25.4.tar.gz"
+  sha256 "6c30bb7ce6ae552b4146ac1c228c18c9e3cf318860ff0fc709cee803406ce900"
+  head "https://github.com/steve8x8/geotoad.git"
+
+  def install
+    mv "geotoad.rb", "geotoad"
+    libexec.install "data", "geotoad", "interface", "lib", "templates"
```

Comment:
Is `libexec.install` the best approach? All of these need to be together in a directory, or it won’t run. Not using `prefix.install` since that causes an error on `audit` due to the `lib`directory (`Non-libraries were installed to …`).